src/azure-cli/azure/cli/command_modules/synapse/manual/_params.py

```diff
@@ -947,7 +947,7 @@ def load_arguments(self, _):
     for scope in ['show', 'create', 'delete']:
         with self.argument_context('synapse managed-private-endpoints ' + scope) as c:
             c.argument('workspace_name', arg_type=workspace_name_arg_type, id_part='name')
-            c.argument('managed_private_endpoint_name', options_list=['--pe-name'], help='The managed private endpoint name.')
```

Comment:
- Hold on, we need discuss how to fix it.
- Regarding this PR, I'm curious what's change for the help message as you add `arg_type=workspace_name_arg_type` for the argument `managed_private_endpoint_name`? I tried, but no change for `--workspace-name`.

Reply:
Currently running `az synapse managed-private-endpoints` command to perform delete operation with only --pe-name fails with below error:
(--workspace-name | --ids) are required

So the `workspace_name_arg_type` parameter should be made **required** here and _not optional._ Also [this](https://docs.microsoft.com/en-us/cli/azure/synapse/managed-private-endpoints?view=azure-cli-latest#az-synapse-managed-private-endpoints-delete) documentation is incorrect.

Reply:
@jsntcy Thanks for looking into this. I shall wait for your reply.

Reply:
We'll discuss if we should fix this issue inside CLI team. 
Even if we need fix it, we should fix it in cli core, but not in specific modules as it's a shared design for all modules (i.e. if user provides `--ids`, like `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/synapse-cli000001/providers/Microsoft.Synapse/workspaces/clitest000003`, then they don't need provide `--workspace-name` as `--ids` contains everything including `--workspace-name`.

Reply:
@jsntcy Thanks for the update. Please keep me posted about the progress, so that I can update the OP of this issue https://github.com/Azure/azure-cli/issues/23562 and set the expectations.